### DISCUSSION TITLE
Target resource types

### DISCUSSION THREAD

**Discussion by delpic** [kedacore/keda#2396] [Q&A / Need Help]
Hello,

I had asked a question on the [Keda Slack channel](https://kubernetes.slack.com/archives/CKZJ36A5D/p1638566679296400) and wanted to followup here. We are using Spinnaker to deploy our applications to Kubernetes and we are using `ReplicaSet`s instead of `Deployment`s so that Spinnaker handles the [rollout strategy](https://spinnaker.io/docs/guides/user/kubernetes-v2/traffic-management/#you-must-use-replica-sets).

I understand that `Deployment` types are the preferred way to go, but, are there any future plans to support `ReplicaSet` types as a target resource?

Thank you.

**Comment by zroubalik** [comment]
Any resource that supports `/scale` subresource can be currently scaled by KEDA. If I am not mistaken, `ReplicaSet` implements `/scale` so it should work for you.  Though I agree with the comments in the Slack thread, that you should definitely use Deployments.
